app/controllers/placements/providers/users_controller.rb

```diff
@@ -8,4 +10,8 @@ def set_organisation
   def redirect_to_index
     redirect_to placements_provider_users_path(@organisation)
   end
+
+  def set_provider
```

Comment:
Isn't this session already set at this point? And should this change the `set_organisation` method?